prod/westeurope/support/functions_backoffice/function_app/terragrunt.hcl

```diff
@@ -78,6 +78,12 @@ inputs = {
 
     #SLOT_TASK_HUBNAME = "ProductionTaskHub"
 
+    # TODO
+    IN_MEMORY_CACHE_TTL = 300
+
+
+    ADB2C_ADMIN_GROUP_NAME = "Admin"
```

Comment:
@BurnedMarshal the group I've created is Admin ( with a in upper case)

Reply:
lgtm like users group name